tests/expected-output-mig-mixed.txt

```diff
@@ -4,6 +4,13 @@ nvidia\.com\/cuda\.driver\.minor=[0-9]+
 nvidia\.com\/cuda\.driver\.rev=[0-9]*
 nvidia\.com\/cuda\.runtime\.major=[0-9]+
 nvidia\.com\/cuda\.runtime\.minor=[0-9]+
+nvidia\.com\/cuda\.driver-version\.major=[0-9]+
+nvidia\.com\/cuda\.driver-version\.minor=[0-9]+
+nvidia\.com\/cuda\.driver-version\.revision=[0-9]*
+nvidia\.com\/cuda\.runtime-version\.major=[0-9]+
+nvidia\.com\/cuda\.runtime-version\.minor=[0-9]+
+nvidia\.com\/cuda\.driver-version\.full=([0-9]+)(.[0-9]+)(.[0-9]+)
```

Comment:
nit: let's keep these sorted.

Reply:
For sure, will do

Reply:
Sorr ... I just mean the ones you added.

Reply:
Got it. Will update

Reply:
Done